test/e2e/upgrades/storage/persistent_volumes.go

```diff
@@ -94,6 +91,11 @@ func (t *PersistentVolumeUpgradeTest) Teardown(f *framework.Framework) {
 	}
 }
 
+func (t *PersistentVolumeUpgradeTest) Skip(upgCtx upgrades.UpgradeContext) bool {
+	// TODO: generalize this to other providers
+	return !framework.ProviderIs("gce", "gke")
+}
+
```

Comment:
Thanks for your contribution! @adamwg 
Can you explain the reason for this revision?
Because I can't find out where this function is being called.

Reply:
`Skip` gets called by the [upgrade test framework here](https://github.com/adamwg/kubernetes/blob/local-upgrade-tests/test/e2e/lifecycle/cluster_upgrade.go#L388), to determine whether a particular test should be skipped. Previously this test was [being skipped from the Setup function](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/upgrades/storage/persistent_volumes.go#L55) if the provider wasn't gce or gke, but that caused the entire upgrade suite to be skipped, not just this portion.